tests/observability/metrics/conftest.py

```diff
@@ -1170,3 +1172,36 @@ def deleted_vmi(running_metric_vm):
 @pytest.fixture()
 def deleted_windows_vmi(windows_vm_for_test):
     windows_vm_for_test.delete(wait=True)
+
+
+@pytest.fixture()
+def application_aware_resource_quota_creation_timestamp(application_aware_resource_quota_scope_module):
+    return application_aware_resource_quota_scope_module.instance.metadata.creationTimestamp
+
+
+@pytest.fixture(scope="module")
+def enabled_aaq_in_hco_scope_module(admin_client, hco_namespace, hyperconverged_resource_scope_module):
+    with enable_aaq_in_hco(
+        client=admin_client,
+        hco_namespace=hco_namespace,
+        hyperconverged_resource=hyperconverged_resource_scope_module,
+    ):
+        yield
+
+
+@pytest.fixture()
+def aaq_resource_hard_limit_and_used(application_aware_resource_quota_scope_module):
+    application_aware_resource_quota_instance = application_aware_resource_quota_scope_module.instance
+    resource_hard_limit = application_aware_resource_quota_instance.spec.hard
+    resource_used = application_aware_resource_quota_instance.status.used
+    return resource_hard_limit, resource_used
+
+
+@pytest.fixture(scope="module")
+def application_aware_resource_quota_scope_module(namespace):
+    with ApplicationAwareResourceQuota(
+        name="application-aware-resource-quota-for-aaq-test",
+        namespace=namespace.name,
+        hard=ARQ_QUOTA_HARD_SPEC,
```

Comment:
just a nit, could you please reorder fixtures

enabled_aaq_in_hco_scope_module
application_aware_resource_quota_scope_module
application_aware_resource_quota_creation_timestamp
aaq_resource_hard_limit_and_used